### ISSUE TITLE
partical fix for serializer (Keras 1.1.1)

### ISSUE THREAD

**Issue by windmaple** [scienceai/neocortex#13]
I spent some time fixing up the serializer. Fully connected layer/relu/softmax now works w/ models exported from Keras 1.1.1. I did not attempt other layers, but the patch should be a good starting point.

Seems the owner is no longer active. But here is the pull request in case anybody wants to save some time.

https://github.com/scienceai/neocortex/pull/12

**Comment by brianrusso** [comment]
Worth pointing out that since this looks abandoned; kerasjs may be a better alternative which while still having some issues; is more active.

https://github.com/transcranial/keras-js

**Comment by windmaple** [comment]
Agreed. When I looked at keras-js, it didn't support NPM, so ....
